sdk/appcenter/src/test/java/com/microsoft/appcenter/utils/context/AuthTokenContextTest.java

```diff
@@ -97,19 +99,23 @@ public void setAuthTokenTest() {
 
         /* Mock context listener. */
         AuthTokenContext.Listener mockListener = spy(AbstractTokenContextListener.class);
+        String accountId = UUIDUtils.randomUUID().toString();
+        String homeAccountId = accountId + "-other_user_information";
 
         /* Set new auth token. */
         mAuthTokenContext.addListener(mockListener);
-        mAuthTokenContext.setAuthToken("42", "mock-user", mock(Date.class));
-        mAuthTokenContext.setAuthToken(AUTH_TOKEN, "mock-user", mock(Date.class));
+        mAuthTokenContext.setAuthToken("42", homeAccountId, mock(Date.class));
+        mAuthTokenContext.setAuthToken(AUTH_TOKEN, homeAccountId, mock(Date.class));
 
         /* Verify the value is stored. */
         verifyStatic(times(2));
         SharedPreferencesManager.putString(eq(PREFERENCE_KEY_TOKEN_HISTORY), anyString());
 
         /* Verify that listener is called. */
         verify(mockListener, times(2)).onNewAuthToken(notNull(String.class));
-        verify(mockListener, times(1)).onNewUser(notNull(String.class));
+        ArgumentCaptor<UserInformation> captorArg = ArgumentCaptor.forClass(UserInformation.class);
+        verify(mockListener).onNewUser(captorArg.capture());
+        assertEquals(accountId, captorArg.getValue().getAccountId());
```

Comment:
Ok but `assertNotNull(captorArg.getValue())` should be done before this line also.